pkgs/applications/version-management/gitlab/gitaly/default.nix

```diff
@@ -16,15 +17,24 @@ let
         };
       };
   };
+  libgit2_0_27 = libgit2.overrideAttrs (oldAttrs: rec {
```

Comment:
Can you add a note about this fix in the commit description?

Reply:
Sure!

Reply:
Should be fixed in my latest push.